BackofficeBundle/Resources/translations/messages.en.yml

```diff
@@ -38,6 +38,8 @@ open_orchestra_backoffice:
         swchoff:
             on: YES
             off: NO
+        javascript:
+          error: An error occurs during form submission!
```

Comment:
I think we should find a kind of sentence saying that it's not a server error but a client one